util/openssl-update-copyright

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+#
+# Copyright 2015-2018 The OpenSSL Project Authors. All Rights Reserved.
```

Comment:
Fun fact: wrong copyright year in the copyright script: It should be `2018` instead of `2015-2018`. (Header was copy&pasted from `openssl-format-source`). Will be fixed.